src/utils/timeUtils.ts

```diff
@@ -0,0 +1,31 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+/**
+ * Get a string in the `#m #s` format. E.g. `2m 34s`.
+ *
+ * @param start Starting Date
+ * @param end Ending Date
+ *
+ * **NOTE:** If the difference between `start` and `end` is negative, `end` will be treated as the current time.
+ */
+// tslint:disable-next-line: export-name
+export function getTimeElapsedString(start: Date, end: Date): string {
```

Comment:
I think you can probably replace this with this https://momentjs.com/docs/#/displaying/difference/

Reply:
I can't see any way to cleanly format a moment object to `2m 31s`.

It seems like it's a missing feature that people have wanted for a while and is still not implemented in moment: https://github.com/moment/moment/issues/463

I've looked at both
* https://momentjs.com/docs/#/displaying/format/
* and https://momentjs.com/docs/#/durations/humanize/

And neither can do the job.

Reply:
IMO it feels like you're trying to pack too much info into the description. This piece feels the least important relative to it's code complexity and I would get rid of it.